opentreemap/treemap/search.py

```diff
@@ -406,7 +411,10 @@ def _parse_dict_value_for_mapping(mapping, valuesdict, field=None):
 
 
 def _parse_predicate_pair(key, value, mapping):
-    model, search_key = _parse_predicate_key(key, mapping)
+    try:
+        model, search_key = _parse_predicate_key(key, mapping)
+    except ModelParseException:
+        return FilterContext()
```

Comment:
Consider adding a comment here explaining why this is being done.

Also, you may be able to be a little more careful here, so that we don't swallow exceptions for simple typos:
```python
try:
    model, search_key = _parse_predicate_key(key, mapping)
except ModelParseException:
    if _is_udf(key):
        # We should return no results for collection UDF searches on unrelated models
        return FilterContext()
    else:
        raise
```